service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseManagementOrderProgressionService.java

```diff
@@ -97,4 +75,14 @@ private void progressActionCaseManagementOrder(CaseDetails caseDetails, CaseData
                 break;
         }
     }
+
+    private List<Element<CaseManagementOrder>> addOrderToList(CaseData caseData) {
+        List<Element<CaseManagementOrder>> orders = caseData.getServedCaseManagementOrders();
+        orders.add(0, Element.<CaseManagementOrder>builder()
```

Comment:
Is business happy with the fact we are adding to the beginning of the list and we will have CMO 1 as the latest one? I remember some conversation and they preferred to retain CMO1 as first CMO :(

Reply:
A/C states that the latest CMO is always at the top. I am happy to add some labels at a future date? But I am going to urge any changes be put into separate tech debt / stories to unblock me.